src/backend/utils/storage.js

```diff
@@ -58,6 +61,13 @@ module.exports = {
 
   isInvalid: id => redis.exists(createInvalidFeedKey(id)),
 
+  setDelayedFeed: (id, seconds) => {
+    const key = createDelayedFeedKey(id);
+    redis.set(key, seconds, 1);
```

Comment:
I think we need a `return` here (line 66), and also in the `setInvalidFeed` above.  We need to return the Promise that this will give us.

Reply:
Done: https://github.com/Seneca-CDOT/telescope/pull/726/commits/850b45fd82e7ffcf6bcc1c91ecc3b4b9e7efd949